src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -713,10 +744,7 @@
             if (indicesBuilder != null)
             {
                 // spill the array if there were await expressions in the indices
-                if (builder == null)
-                {
-                    builder = new BoundSpillSequenceBuilder(indicesBuilder.Syntax);
-                }
```

Comment:
>builder ??= new BoundSpillSequenceBuilder(indicesBuilder.Syntax); [](http://example.com/codeflow?start=16&length=65)

Please revert changes like that. We do not welcome style changes in general, and do not mix pure refactorings with functional changes in the same PR.